example_client/stateful/asr_demo/README.md

```diff
@@ -133,3 +133,63 @@ When the command finishes successfully you should see the `txt` file in the same
 cat /opt/workspace/sample.wav.txt
 /opt/workspace/sample.wav today we have a very nice weather
 ```
+
+### 7. Automatic speech recognition
+You can also run the live-demo.py client on windows machine to record wav files in real time with your microphone and send them to a ssh enabled server with mentioned kaldi and ovms containers setup.
+
+On server side run the instructions steps from 1 to 5 but instead of commands in step 6, run the following commands:
+```
+export DATA_DIR=$HOME/asr_demo/data
+mkdir -p DATA_DIR
+```
+
+Start kaldi container built in the step 2 in interactive mode:
+```
+docker run --rm -it --network="host" -v $DATA_DIR:/opt/data kaldi:latest bash
+```
+
+Run speech recognition loop:
+```
+/opt/model_server/example_client/stateful/asr_demo/run_auto.sh localhost 9000
```

Comment:
What does it do? How is it different than run.sh? Could you explain the flow of that script beneath the command?